### DISCUSSION TITLE
fitting trigger for (() => {})

### DISCUSSION THREAD

**Discussion by reaseno** [deinsoftware/vscode-arrow-snippets#14] [Q&A]
Hi, I often need this:

```
(() => {

})
```

How can I achieve that? I can't find a fitting trigger 🤷‍♂️

**Comment by equiman** [comment]
There is no shortcut for that, but you can use one of these two options:

- `afii` or `iiaf` and add the `{ }` on cursor position
- write `()` and between use `afee`

**Reply by reaseno** [reply]
Can this maybe be added in a future release?

Thank you for your help.

**Comment by equiman** [comment]
could be, I'm a little busy right now, but I will try to add it soon.
